src/main/scala/dsls/regex/RegularExpression.scala

```diff
@@ -34,3 +56,16 @@ case class Concat(val left: RegularExpression, val right: RegularExpression)
  *  expression
  */
 case class Star(val expression: RegularExpression) extends RegularExpression
+
+/** Ensure that we can perform implicit conversions 
+ *  from chars and strings to our RegularExpressions
+ */
+object RegularExpressionImplicit {
```

Comment:
You can call this object `RegularExpression` (the same name as the abstract class). Then the user can write `import RegularExpression._` instead of `import RegularExpressionImplicit._`, which might be more intuitive if the user doesn't know about implicit conversions.